lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -206,17 +206,67 @@ defmodule Bamboo.SMTPAdapter do
   end
 
   defp add_subject(body, %Bamboo.Email{subject: subject}) do
-    add_smtp_header_line(body, :subject, rfc822_encode(subject))
-  end
-
-  defp rfc822_encode(content) do
-    "=?UTF-8?B?#{Base.encode64(content)}?="
+    add_smtp_header_line(body, :subject, rfc2047_encode(subject))
   end
 
   defp rfc2231_encode(content) do
     "UTF-8''#{URI.encode(content)}"
   end
 
+  def rfc5322_encode(content) do
```

Comment:
I couldn't find anything on this RFC that was explicitly forbidding quotes and backslashes. Can you point me to it?

Reply:
Ah, nevermind. I see that you linked from the PR description :+1: